CRM/Contact/Form/Edit/Demographics.php

```diff
@@ -45,10 +45,10 @@ class CRM_Contact_Form_Edit_Demographics {
   public static function buildQuickForm(&$form) {
     $form->addField('gender_id', array('entity' => 'contact', 'type' => 'Radio', 'allowClear' => TRUE));
 
-    $form->addField('birth_date', array('entity' => 'contact'), FALSE, FALSE);
+    $form->addField('birth_date', array('entity' => 'contact'), FALSE, array(), FALSE);
```

Comment:
The legacyDateParam is v recent & transitional so I know these are the only fields using it